src/components/fields/NullField.js

```diff
@@ -0,0 +1,20 @@
+import { Component } from "react";
+import PropTypes from "prop-types";
+
+class NullField extends Component {
+  componentDidMount() {
+    this.props.onChange(null);
```

Comment:
Perhaps it would be better to call onChange only if `props.value === undefined`?

Reply:
Agreed, that behaviour might come in handy.